annotationPipeline/src/main/java/org/cbioportal/annotation/pipeline/MutationRecordReader.java

```diff
@@ -105,33 +110,41 @@ public void handleLine(String line) {
             throw new ItemStreamException(e);
         }
         reader.close();
-        
+
         for(MutationRecord record : mutationRecords) {
             // save variant details for logging
-            String variantDetails = "(chr,start,end,ref,alt)= (" + record.getChromosome() + "," + record.getStart_Position() + ","
-                    + record.getEnd_Position() + "," + record.getReference_Allele() + "," + record.getTumor_Seq_Allele2() + ")";
-            
+            String variantDetails = "(sampleId, chr,start,end,ref,alt,url)= (" + record.getTumor_Sample_Barcode() +  record.getChromosome() + "," + record.getStart_Position() + ","
```

Comment:
space after first comma, no spaces after other commas (in line intro/prefix). Then in the filled-in data there is no comma added between the first and second values.